pkg/utils/utils.go

```diff
@@ -460,3 +460,17 @@ func GetClusterFlavor(clientset *kubernetes.Clientset) (ClusterFlavor, error) {
 	// Did not match any search criteria. Unknown cluster flavor.
 	return Unknown, errors.New("GetClusterFlavor: Failed to identify cluster flavor")
 }
+
+func CreateRepositoryFromBackupRepository(backupRepository *backupdriverapi.BackupRepository, logger logrus.FieldLogger) (*s3repository.ProtectedEntityTypeManager, error) {
```

Comment:
Are we really creating a new repo here or just get a repo?  Should we call this GetRepositoryFromBackupRepository?

Reply:
Actually here GetS3PETMFromParamsMap will be called with BackupRepsitory RepositoryParameters, each time it will new S3RepositoryProtectedEntityTypeManager. Change the name to GetRepositoryFromBackupRepository to keep consistent with GetS3PETMFromParamsMap.

Reply:
The function name, CreateRepositoryFromBackupRepository, is really confusing here as we don't actually create anything. Maybe we can consider something like, Get, Retrieve, etc.